vaadin-upload-flow-parent/vaadin-upload-flow/src/test/java/com/vaadin/flow/component/upload/tests/UploadDropLabelTest.java

```diff
@@ -3,26 +3,17 @@
 import com.vaadin.flow.component.UI;
 import com.vaadin.flow.component.html.Span;
 import com.vaadin.flow.component.upload.Upload;
-import com.vaadin.flow.server.VaadinSession;
 import net.jcip.annotations.NotThreadSafe;
-import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mockito;
 
 @NotThreadSafe
 public class UploadDropLabelTest {
-    @Before
-    public void setup() {
-        UI ui = new UI();
-        UI.setCurrent(ui);
-        VaadinSession session = Mockito.mock(VaadinSession.class);
-        ui.getInternals().setSession(session);
-    }
```

Comment:
There are several other components that set up the UI in `@Before`. 

- `DateTimePickerTest` 
- `NotificationTest`
- `TimePickerLocaleTest`
- and so on...

Are they also flaky?